README.md

```diff
@@ -1,5 +1,11 @@
 # Wagtail Headless Preview
 
+## Overview
+
+With wagtail as the backend, and a separate app for the frontend (for example a single page react app), the editor will still need to see how the page will look when published. The frontend is no longer within wagtail's control, so the preview data/markup needs to be exposed to the frontend app.
```

Comment:
wagtail  ➜ Wagtail.

I am thinking something along the lines of:
"This package enables previews for Wagtail pages when used in a headless setup by routing the preview the specified frontend URL.

With Wagtail as the backend, and a separate app for the frontend (for example, a single page React app), editors are no longer able to preview their changes out of the box because he frontend is no longer within Wagtail's direct control, so the preview data/markup needs to be exposed to the frontend app."